vllm/platforms/interface.py

```diff
@@ -27,16 +35,64 @@ def is_tpu(self) -> bool:
     def is_cpu(self) -> bool:
         return self._enum == PlatformEnum.CPU
 
-    @staticmethod
-    def get_device_capability(device_id: int = 0) -> Optional[Tuple[int, int]]:
+    @classmethod
+    def is_cuda_available(cls, device_id: int = 0) -> bool:
+        return cls.get_device_capability(device_id=device_id) is not None
```

Comment:
why do we need this?

Reply:
It is to avoid calling `torch.cuda.is_available` (but see the note above)